build/opt.go

```diff
@@ -440,6 +441,11 @@ func loadInputs(ctx context.Context, d *driver.DriverHandle, inp Inputs, pw prog
 
 	if inp.DockerfileInline != "" {
 		dockerfileReader = io.NopCloser(strings.NewReader(inp.DockerfileInline))
+		dockerfileSrcName = "inline"
+	} else if inp.DockerfilePath == "-" {
+		dockerfileSrcName = "stdin"
```

Comment:
What will be the output for `inline` and `stdin` case? I guess it should be omitted like:

```
Command 'copy' should match the case of the command majority (uppercase)
--------------------
   1 |     FROM alpine
   2 | >>> copy ./tmp /tmp
   3 |
   4 |
--------------------
```

Something like this would be confusing:

```
Command 'copy' should match the case of the command majority (uppercase)
inline:2
--------------------
   1 |     FROM alpine
   2 | >>> copy ./tmp /tmp
   3 |
   4 |
--------------------
```

If someone has a Dockerfile named `inline`

Reply:
Removing the line entirely requires either a `buildkit` PR or refactoring how `buildx` is printing warnings to effectively duplicate that functionality. I'm not opposed, but if that's the goal then it might be best to merge this and then make a corresponding issue in `buildkit` for the follow-up work.

I was trying to model what @tonistiigi had suggested, here. For MY part, I think that ambiguity around two very specifically named Dockerfiles (in this case, `stdin` and `inline`) is less confusing than simply not showing anything in these cases. That being said, it could be updated to something a little more clear and even less likely to be a filename?

```
Command 'copy' should match the case of the command majority (uppercase)
Inline Dockerfile:2
--------------------
...
```

```
Command 'copy' should match the case of the command majority (uppercase)
Dockerfile (inline):2
--------------------
...
```

Reply:
> That being said, it could be updated to something a little more clear and even less likely to be a filename?

Yes that sounds good, let's check that for a follow-up then